doc/source/whatsnew/v1.3.0.rst

```diff
@@ -285,6 +285,7 @@ Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 - Bug in :meth:`SeriesGroupBy.value_counts` where unobserved categories in a grouped categorical series were not tallied (:issue:`38672`)
+- Bug in :meth:`.GroupBy.indices` would contain non-existent indices when null values were present in the groupby keys (:issue:`9304`)
```

Comment:
this won't render i think can just say DataFrameGroupBy.indices

Reply:
This does render:

![image](https://user-images.githubusercontent.com/45562402/103425677-fea4a700-4b81-11eb-81da-ef08b03ba918.png)